libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Util/SFManagedPreferences.m

```diff
@@ -94,6 +95,13 @@ - (id)init {
     return self;
 }
 
+- (void)initializeManagedPreferences {
+    self.rawPreferences = [[NSUserDefaults msdkUserDefaults] dictionaryForKey:kManagedConfigurationKey];
+    if (self.rawPreferences) {
```

Comment:
But ultimately, @bhariharan, @trooper2013, and @wmathurin I think these app feature loggings would be better managed through an intermediary, that basically fronted the storage of these "features" (e.g. in `NSUserDefaults`) on the one hand, and the retrieval from e.g. `SalesforceSDKManager` on the other.  That way, these singleton state issues would never come into play.  My $0.02.